examples/multiagent/iql.py

```diff
@@ -0,0 +1,283 @@
+import time
+
+import torch
+
+import wandb
```

Comment:
Let's use our wandb logger.
Better: let's make the logger configurable via a `backend` arg in the config